services/frontend/src/styles/_mixins.scss

```diff
@@ -29,6 +30,15 @@ $default-percentage: 5%;
     }
 }
 
+@mixin hover-box-shadow() {
+    transition: box-shadow 2 ease-out;
+    box-shadow: 0;
```

Comment:
You should remove this since it's invalid.

Reply:
True, removed the:
```js
box-shadow: 0
```